src/components/IdentityView/IdentityView.tsx

```diff
@@ -79,45 +81,49 @@ class IdentityView extends React.Component<Props, State> {
         <div className="attributes">
           <div>
             <label>Alias</label>
-            <div>{myIdentity.metaData.name}</div>
+            <div>{metaData.name}</div>
           </div>
           <div>
             <label>Phrase</label>
-            <div>{myIdentity.phrase}</div>
+            <div>{phrase}</div>
           </div>
           <div>
             <label>KILT Address</label>
-            <div>{myIdentity.identity.address}</div>
+            <div>{identity.address}</div>
           </div>
           <div>
             <label>Seed (as hex)</label>
-            <div>{myIdentity.identity.seedAsHex}</div>
+            <div>{identity.seedAsHex}</div>
           </div>
           <div>
             <label>Public Key</label>
-            <div>{myIdentity.identity.signPublicKeyAsHex}</div>
+            <div>{identity.signPublicKeyAsHex}</div>
           </div>
           <div>
             <label>Encryption Public Key</label>
-            <div>{myIdentity.identity.boxPublicKeyAsHex}</div>
+            <div>{identity.boxPublicKeyAsHex}</div>
           </div>
           <div>
             <label>DID</label>
```

Comment:
Your UI looks good!
However DID and DID Document are not the same thing, so the label is not quite right. 
I'd suggest to append `Document` to the `<label>` when the DID Doc is displayed.

Reply:
I have updated it, so it now looks like `DID` - When there isn't a `DID`. But, `DID Document`.

Reply:
![Screenshot 2020-01-23 at 14 32 13](https://user-images.githubusercontent.com/48058984/72988764-32614380-3ded-11ea-8c13-b60141c78187.png)